core/domain/subtopic_validatiors_test.py

```diff
@@ -0,0 +1,846 @@
+# coding: utf-8
```

Comment:
This file has a wrong name: subtopic_validatiors_test.py → subtopic_validators_test.py